src/UpVote.jsx

```diff
@@ -0,0 +1,29 @@
+import { patchVote } from "./api";
+import { useState } from "react";
+
+const UpVote = ({ votes, article_id }) => {
+
+    const [voteChange, setVoteChange] = useState(0);
+
+    const increaseVote = (voteChange) => {
+        setVoteChange((currVote) => currVote + voteChange)
+        patchVote(article_id, voteChange).catch(() => {
+            setVoteChange((currVote) =>
+                currVote - voteChange)
+            alert("Vote failed to submit")
```

Comment:
FYI `alert`'s are synchronous and blocking. It's fine here but as a general rule they should be avoided in the real world as your app will be frozen until it's closed.